docs/docs/getting-started/building-an-image.md

```diff
@@ -0,0 +1,32 @@
+---
+sidebar_position: 3
+title: Build a Kernel
+---
+
+After making any required modifications to your kernel, the project uses
+```shell
+docker buildx bake
```

Comment:
specify version of buildx or link to it from the workflows

Reply:
The buildx version is tied to the Docker version installed. We'll track this in the other conversation